src/librustdoc/html/static/themes/dark.css

```diff
@@ -389,3 +389,8 @@ kbd {
 		background: #f0f0f0;
 	}
 }
+
+#all-types {}
```

Comment:
This ID gets a `background-color` in the light theme; is it valid to leave it blank here? That seems like it will cause some problems at some point. `>_>`

Reply:
It doesn't need to overload the rule at the opposite of `light.css` so I left it empty. But I can update it, as you wish.